### ISSUE TITLE
[Screen Reader-MCIO DCX-O1NLD]: NVDA does not announce the Column header name information when the focus lands on the column headers.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7244]
**Description:**
Users who rely on screen readers will be impacted if the screen reader does not announce the link name and correct information of the column headers. Screen reader users cannot know the headers and cannot get the context of the table clearly.

**Environment (OS, Application, Versions):**
**Test Environment:**
OS Build: Version 21H2(22000.856)
Browser: Microsoft Edge Version 105.0.1343.27 (Official build) (64-bit)
Screen reader: Non-Visual Desktop Access (NVDA) (2022.2)

**AT (include version number):**
NVDA (2022.2)

**Pre-Requisites (if any):**
Turn on NVDA (Ctrl + Alt+ N)

**Repro Steps:**
1. Open [Angular Grid | Excel Style Filtering | Infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering)
2. Grids excel style filtering pen page will open.
3. Reach to the main landmark content where the ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5.  Verify the issue that NVDA announces column header name or not.

**Actual Results:**
NVDA announces incorrect name of the Column header when the focus lands on the column headers.
NVDA is announcing all the column headers name at once and repeatedly announcing clickable when the focus lands on the 'Product Name' column header.

**Expected Results:**
NVDA should announce the Column header name information correctly when the focus lands on the column headers.
NVDA should announce as 'Product Name Column header 1 of 5' when the focus lands on the 'Product Name' column header.

**Additional Information:**
Same Issue Reproes with Narrator and JAWS.
The narrator is announcing as 'group' when the focus lands on the column header.
JAWS remains silent when the focus lands on the column header.
Same Issue Reproes with all the column headers of the tables present throughout the application.

**Attachment:**
https://user-images.githubusercontent.com/93735775/188449334-31748d8d-168e-4ad1-82a1-9acee6000e62.mp4

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
